ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -66,6 +67,24 @@ ORK_AVAILABLE_DECL
  */
 - (void)passcodeViewControllerDidCancel:(UIViewController *)viewController;
 
+/*
+ * Defaults to Localized "Forgot Password?" text
+ * @return the text of the forgot passcode button
+ */
+- (NSString*)passcodeViewControllerTextForForgotPasscode:(UIViewController *)viewController;
+
+/*
+ * Defaults to Localized standard Apple blue tint color
+ * @return the tint color of the forgot passcode button
+ */
+- (UIColor*)passcodeViewControllerTintColorForForgotPasscode:(UIViewController *)viewController;
```

Comment:
Ohhh! now I see what you are talking about with the delegate methods.  Thanks for taking the lead on changing these yourself.  Sometimes it is just faster that way.